packages/block-library/src/columns/edit.js

```diff
@@ -69,6 +75,10 @@ function ColumnsEditContainer( {
 
 	const blockProps = useBlockProps( {
 		className: classes,
+		style: {
+			'--gutter-size':
```

Comment:
I wonder if we should make the namespacing clearer here, eg. --columns-block-gutter-size, just to make it clearer when debugging in dev tools, etc. what this var is related to ?

Reply:
Given the CSS var was not set on the `:root` selector and only via the `.wp-block-columns` scope I felt it was clear enough where this gutter size was coming from and its purpose. 

It doesn't hurt to improve clarity of course so I've made the suggested change.  Thanks 🙂